chainer/function_hook.py

```diff
@@ -76,14 +84,24 @@ class FunctionHook(object):
        as a thread local object. So, function hooks registered
        are different depending on threads.
 
-    The other one is to register directly to
+    The other one is to register it directly to
     :class:`~chainer.FunctionNode` object with
-    :meth:`~chainer.Function.add_hook` method.
+    :meth:`~chainer.FunctionNode.add_hook` method.
     Function hooks registered in this way can be removed by
```

Comment:
Nitpick, actually not directly related to this PR...
registered in this way ... by -> registered this way ... by the